pgusinit/pgusinit.c

```diff
@@ -19,25 +19,26 @@ typedef enum {
 
 
 void banner(void) {
-    printf("PicoGUSinit v1.2.0\n");
+    printf("PicoGUSinit v1.2.1\n");
     printf("(c) 2023 Ian Scott - licensed under the GNU GPL v2\n\n");
 }
 
 
-void usage(void) {
+void usage(char *argv0) {
     // Max line length @ 80 chars:
     //              "................................................................................\n"
-    fprintf(stderr, "usage: pgusinit [/?] | [/b x] [/a n] [/d n] | [/f fw.uf2]\n");
+    fprintf(stderr, "Usage:\n");
+    fprintf(stderr, "  %s [/?] | [/p x] [/a n] [/d n] | [/f fw.uf2]\n", argv0);
     fprintf(stderr, "\n");
     fprintf(stderr, "    /?   - show this message\n");
     fprintf(stderr, "    /f fw.uf2 - Program the PicoGUS with the firmware file fw.uf2.\n");
     fprintf(stderr, "AdLib, MPU-401, Tandy, CMS modes only:\n");
-    fprintf(stderr, "    /b x - set the base address of the emulated card. Defaults:\n");
+    fprintf(stderr, "    /p x - set the (hex) base address of the emulated card. Defaults:\n");
     fprintf(stderr, "           AdLib: 388; MPU-401: 330; Tandy: 2C0; CMS: 220\n");
     fprintf(stderr, "GUS mode only:\n");
     fprintf(stderr, "    /a n - set audio buffer to n samples. Default: 16, Min: 8, Max: 256\n");
     fprintf(stderr, "           (tweaking this can help programs that hang or have audio glitches)\n");
-    fprintf(stderr, "    /d n - force DMA interval to n �s. Default: 0, Min: 1, Max: 256\n");
+    fprintf(stderr, "    /d n - force DMA interval to n �s. Default: 0, Min: 0, Max: 256\n");

```

Comment:
Agree with @kmeaw - this is `outb()`ed as-is and is interpreted in the PicoGUS firmware as-is so the max value is 255.
```suggestion
    fprintf(stderr, "    /d n - force DMA interval to n �s. Default: 0, Min: 0, Max: 255\n");

```

Reply:
did the requested changes